src/commands/delete.js

```diff
@@ -0,0 +1,53 @@
+const utils = require('../utils');
+
+const _delete = (context, appOrVersion, version) => {
+  if (appOrVersion === 'version' && !version) {
+    const message = 'Error: No version - provide a version like "1.0.0"...';
+    return Promise.reject(new Error(message));
+  }
+  return utils.checkCredentials()
+    .then(() => utils.getLinkedApp())
+    .then((app) => {
+      const deletePrepMessage = (appOrVersion === 'version')
+        ? `Preparing to delete version ${version} of your app "${app.title}".\n`
+        : 'Preparing to delete all versions of your app "${app.title}".\n';
+      context.line(deletePrepMessage);
+      const url = appOrVersion === 'version' ? `/apps/${app.id}/versions/${version}` : `/apps/${app.id}`;
+      const deleteMessage = appOrVersion === 'version' ? `Deleting version ${version}` : 'Deleting app';
+      utils.printStarting(deleteMessage);
+      return utils.callAPI(url, {
+        method: 'DELETE',
+      });
+    })
+    .then(() => {
+      utils.printDone();
+      context.line('  Deletion successful!\n');
+    });
+};
+_delete.argsSpec = [
```

Comment:
Food for thought (non-blocker) maybe we would (in the future) create a `command` util function that does:

```js
module.exports = command({
     argsSpec: [...],
    ...
    example: '...',
    run(context, appOrVersion, version) {

    }
});
```

It would just do all of the manual `_delete.example = ` stuff and return the run function as the export.